docs/source/explore/get-started/install.rst

```diff
@@ -11,30 +11,53 @@ Vircadia has two different installers. The *Client Installer* comes with everyth
 Minimum System Requirements
 --------------------------------
 
-In order to run Vircadia in either VR or Desktop mode, ensure that your computer meets these minimum system requirements: 
+In order to run Vircadia, ensure that your computer meets these minimum system requirements:
 
-* Windows 10, 64-bit or MacOS High Sierra (10.13)
-* 8GB+ RAM i5
-* A wide range of Nvidia, AMD, and Intel graphics cards are supported
+* Operating system:
+
+  * Windows 8.1, 64-bit minimum
+  * Windows 10, 64-bit recommended
+  * MacOS High Sierra (10.13) or newer
```

Comment:
I don't think we should put minimum in front of this, as it is a "requirement" and newer mac version aren't more recommended as High Sierra

Reply:
Mayhaps then the word "Required"? I wonder if any other versions work about on par with that. Otherwise maybe we should make clear that we really only support 10.13 and that newer *should* work...